test/src/test/java/org/corfudb/test/benchmark/AbstractCorfuBenchmark.java

```diff
@@ -0,0 +1,52 @@
+package org.corfudb.test.benchmark;
+
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Test;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+import org.openjdk.jmh.runner.options.TimeValue;
+
+/**
+ * Junit bridge for jmh.
+ *
+ * To implement your own benchmark class, subclass (extend) this class and provide your own state by
+ * extending {@link CorfuBenchmarkState} as a static inner class.
+ *
+ * Annotate your benchmark functions with {@link org.openjdk.jmh.annotations.Benchmark}.
+ * This will mark them for being run when the {@link this#runBenchmarks()} junit test is run.
+ *
+ * Created by zlokhandwala on 5/3/19.
+ */
+public abstract class AbstractCorfuBenchmark {
```

Comment:
Please don't bring unnecessary abstractions. This class should be refactored and shouldn't exists